bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BXMLItem.java

```diff
@@ -263,15 +263,6 @@ public void setAttribute(String localName, String namespaceUri, String prefix, S
             return;
         }
 
-        // Attributes cannot cannot be belong to default namespace. Hence, if the current namespace is the default one,
```

Comment:
We no longer allow default namespace URI of the element to be used as 'xmlns'. 
When user actually use 'xmlns' URI it's handled in prefix handling code at the end of this method.